docs/docs/step-06.md

```diff
@@ -152,6 +153,43 @@ Here, we use a recursive document splitter with a segment size of 100 tokens and
 
 Finally, we trigger the ingestion process and log a message when it's done.
 
+### In-memory embedding store
+
+If you are not able to run Dev Services with Docker or Podman, feel free to use the in-memory
+embedding store provided by LangChain4j.
+
+Add the following dependency to your `pom.xml` file:
+
+```
+<dependency>
+    <groupId>dev.langchain4j</groupId>
+    <artifactId>langchain4j</artifactId>
+</dependency>
+```
+
+And create an `EmbeddingStore` producer in a new class `InMemoryEmbeddingStoreProvider`:
+
+```java
+package dev.langchain4j.quarkus.workshop;
+
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.inmemory.InMemoryEmbeddingStore;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.inject.Produces;
+
+@ApplicationScoped
+public class InMemoryEmbeddingStoreProvider {
+
+    @Produces
+    @ApplicationScoped
+    EmbeddingStore embeddingStore() {
```

Comment:
Let's avoid raw usage of generic types and change it to `EmbeddingStore<TextSegment>` (you'll need to add an import too)

Reply:
Will not work if we won't change all the injections - https://github.com/quarkusio/quarkus-workshop-langchain4j/blob/main/step-06/src/main/java/dev/langchain4j/quarkus/workshop/RagIngestion.java#L35. And this would need to be done in all following steps. Let's skip it for now and fix this later.

Reply:
I see, ok then